.github/workflows/needinfo-stale.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Close old issues with the needinfo label
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close old issues with the needinfo tag
+        uses: dwieeb/needs-reply@v2
```

Comment:
This is not actively developed anymore, but I've seen that you created a pull-request. Should we wait until that's approved or use a fork?